Utils/Dataflow/009_oracleConnector/Oracle2JSON.py

```diff
@@ -143,9 +205,13 @@ def init_offset_storage(config):
             sys.stderr.write('(INFO) No stored offset found.'
                              ' Using initial date.\n')
             current_offset = initial_date
-        elif current_offset < initial_date:
-            sys.stderr.write('(INFO) Stored offset is less then configured'
-                             ' initial date. Using initial date instead.\n')
+        elif not reverse and current_offset < initial_date \
+                or reverse and current_offset > initial_date:
+            sys.stderr.write('(INFO) Stored offset is %s then configured'
+                             ' initial date (%s data loading). Using initial'
+                             ' date instead.\n'
+                             % ('greater' if reverse else 'less',
+                                'reverse' if reverse else 'normal'))
             current_offset = initial_date
```

Comment:
The same situation with `(INFO)`. 
And one more question. Why in the previous block of code it was double quotes and here - single ones?

Reply:
Still no `\n` in the middle of the message, so it will be output as a single line.
Single and double quotes... er... no idea :) 
I am trying to follow some conventions on this but usually fail...:(
The convention was, as far as I remember, the following:
* use single quotes for 'constants' (like dict keys);
* use double quotes for "strings" (like log messages);
* use double quotes when you need to escape single one and vice versa.

Will fix it. Thank you for pointing :)

Reply:
Done.

Also found missed `\n` in a couple of error messages -- also fixed.